### PR TITLE
update endpoint to list all projects for admin user

### LINKED ISSUE #5148: Add get/list project endpoints for admin panel
**User Story**
Add new endpoints to list all the projects for the admin user

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: update endpoint to list all projects for admin user

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #5148

```release-note
update endpoint to list all projects for admin user
```

### CODE REVIEW

File: api/pkg/handler/routes_v1_admin.go [hunk 1]
```diff
@@ -363,3 +369,27 @@ func (r Routing) deleteSeed() http.Handler {
 		r.defaultServerOptions()...,
 	)
 }
+
+// swagger:route GET /api/v1/admin/projects admin adminListProjects
+//
+//     Lists all the projects.
```
Comment (maciaszczykm) [inline api/pkg/handler/routes_v1_admin.go]: Improve the description. What is the difference with listing the project with current endpoint? If it lists projects of all users (no matter if admin is the user or nor) then please state it in the comment.

File: api/pkg/handler/routes_v1_admin.go [hunk 2]
```diff
@@ -363,3 +369,27 @@ func (r Routing) deleteSeed() http.Handler {
...
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       default: errorResponse
+//       200: []Project
+//       401: empty
+//       409: empty
+func (r Routing) adminListProjects() http.Handler {
```
Comment (maciaszczykm) [inline api/pkg/handler/routes_v1_admin.go]: `listProjects`, `listAllProjects(forAdmin?)` would suit better IMO.

Comment (zreigz) [reply api/pkg/handler/routes_v1_admin.go]: done

File: api/pkg/handler/routes_v1_admin.go (hunk 1 above)
Comment (zreigz) [reply api/pkg/handler/routes_v1_admin.go]: done

**Comment by maciaszczykm** [comment]
/lgtm
/hold

Please change the description and the changelog.

**Pull request merged**